libraries/Microsoft.Bot.Builder.Dialogs.Adaptive.Runtime/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -32,34 +32,39 @@ public static class ServiceCollectionExtensions
         /// </summary>
         /// <remark>
         /// The following dependencies are added with TrySingleton so advanced scenarios can override them to customize the runtimne behavior:
-        /// BotFrameworkAuthentication,
-        /// IBot,
-        /// ResourceExplorer,
-        /// LanguagePolicy,
-        /// ChannelServiceHandlerBase,
-        /// IStorage,
-        /// UserState,
-        /// ConversationState,
-        /// SkillConversationIdFactoryBase
-        /// and IBotTelemetryClient.
+        /// <see cref="BotFrameworkAuthentication"/>,
```

Comment:
Looking much nicer with this

Reply:
so you think its a keeper then - I wasn't quite sure!